storage-proofs/src/stacked/proof.rs

```diff
@@ -388,12 +393,30 @@ impl<'a, H: 'static + Hasher, G: 'static + Hasher> StackedDrg<'a, H, G> {
 
         // Generate all store configs that we need based on the
         // cache_path in the specified config.
-        let mut tree_d_config =
-            StoreConfig::from_config(&config, CacheKey::CommDTree.to_string(), None);
-        let mut tree_r_last_config =
-            StoreConfig::from_config(&config, CacheKey::CommRLastTree.to_string(), None);
-        let mut tree_c_config =
-            StoreConfig::from_config(&config, CacheKey::CommCTree.to_string(), None);
+        let mut tree_d_config = StoreConfig::from_config(
+            &config,
+            CacheKey::CommDTree.to_string(),
+            Some(StoreConfig::default_cached_above_base_layer(
+                data.len(),
+                BINARY_ARITY,
```

Comment:
Shouldn't all of the tree arities be matching?

Reply:
no, comm_d will stay a binary tree

Reply:
maybe will change